docs/CodeDevelopment/requirements.md

```diff
@@ -64,6 +93,13 @@
 2. Use STATIC on each non-public function and non-public global to avoid conflicts with other modules.
 3. Use EFIAPI on all public library class functions.
 
+## Source files
+
+Source files should follow the Edk2 coding standards available on [TianoCore.org](https://www.tianocore.org/) website.  
+
+Please include the appropriate copyright messages in the file headers, and Spdx License
+Identifiers are appreciated (<https://spdx.org/licenses/>).
```

Comment:
I would suggest making the spdx a requirement/demand.  I also would make this a list rather than long sentence to match others.

Reply:
Fixed in latest.